region/info.go

```diff
@@ -373,18 +376,20 @@ func (i *info) MarshalJSON() ([]byte, error) {
 		StopKey         string
 		ContextInstance string
 		Err             string
+		ClientPtr       string
 		Client          string
 		Available       bool
 	}{
 		Id:              i.id,
-		Namespace:       string(i.namespace),
-		Table:           string(i.table),
-		Name:            string(i.name),
-		StartKey:        string(i.startKey),
-		StopKey:         string(i.stopKey),
+		Namespace:       fmt.Sprintf("%q", i.namespace),
```

Comment:
I'd rather we use `strconv.QuoteToASCII`, because it's a little faster than going through `Sprintf`. And I think it deals better with unicode for our uses, ie. it should print out each byte instead of the unicode character.

One step better would to be write our own escape function that did exactly what we wanted, escaping non-ASCII characters rather than quoting the string. Not fully tested, but I think it works:
```
func escapeToASCII(in []byte) string {
	size := len(in)
	for _, c := range in {
		if c <= 0x20 || c >= 0x7f {
			size += 3
		} else if c == '\\' {
			size++
		}
	}
	const hexChars = "0123456789abcdef"
	out := make([]byte, 0, size)
	for _, c := range in {
		if c <= 0x20 || c >= 0x7f {
			out = append(out, '\\', 'x', hexChars[c>>4], hexChars[c&0xf])
		} else if c == '\\' {
			out = append(out, '\\', '\\')
		} else {
			out = append(out, c)
		}
	}
	return string(out)
}
```

Reply:
I used `strconv.QuoteToASCII` here: 
https://github.com/tsuna/gohbase/compare/1448caeca1d21bfcdc19407fee7aa1e35b61a5de..b49d419e5caad70bac8dfade1468f6e788c57c15

We could add a library as a follow up to add escapeToASCII, as well as the testing